index.js

```diff
@@ -4,14 +4,13 @@ import play from './Commands/Play.js'
 import help from './Commands/Help.js'
 import list from './Commands/List.js'
 
+//setup and config 
 const client = new Discord.Client();
 dotenv.config();
 
-const { BOTPREFIX } = process.env;
-const { TOKEN } = process.env;
+const { BOTPREFIX, TOKEN} = process.env;
```

Comment:
nice update. add a space after token. Do you have a linter set up in your code editor?

Reply:
I do, I'm surprised this didn't error, my rules are imported from the Discord docs, I'll check.